Sources/MongoSwift/MongoCursor.swift

```diff
@@ -118,7 +118,13 @@ public class MongoCursor<T: Codable>: Sequence, IteratorProtocol {
     public func next() -> T? {
         do {
             let operation = NextOperation(cursor: self)
-            let out = try operation.execute()
+            // TODO SWIFT-374: we temporarily use a fake connection here. eventually MongoCursor will store its source
```

Comment:
(I deferred this til a future PR as it will require all the methods returning cursors to start checking out connections and I think the diff will be somewhat significant)